kubernetes/zenko/disruption-budget.yaml

```diff
@@ -0,0 +1,19 @@
+## IMPORTANT NOTE: These are the default values for setting an acceptable
+## amount of unavailable pods before a service becomes unavailable. However
+## these values are immutable after first install so please take into
+## consideration when selecting the appropriate numbers for your cluster
+## size and preferences. While all the options listed are default values,
+## this file can be changed and applied at chart installation time by
+## specifying `-f disruption-budget.yaml`.
```

Comment:
This file has multiple `podDisruptionBudgets`, so it would look better if it's pluralized: `disruption-budgets.yaml`. Also, why not have the `pod-` prefix?

Reply:
I left the "pod" out for conciseness but can easily add it. I'll update the name as well.